content/docs/rfcs/13/README.md

```diff
@@ -42,11 +42,11 @@ message Index {
 }
 
 message PagingInfo {
-  int64 pageSize = 1;
+  uint64 pageSize = 1;
   Index cursor = 2;
   enum Direction {
-    FORWARD = 0;
-    BACKWARD = 1;
+    FORWARD = 1;
+    BACKWARD = 0;
```

Comment:
For `proto3` your `0`-valued enum should be the first in the definition to be backwards compatible with `proto2`, where the default was always the first defined (regardless of value). See https://developers.google.com/protocol-buffers/docs/proto3#enum

Reply:
Thanks! addressed!